app/Extensions/AppDelegate+Workspaces.swift

```diff
@@ -44,9 +45,9 @@ extension AppDelegate {
                 guard let self else { return }
                 didSelect?(navigationController)(workspace)
 
-                let comingSoonController = ComingSoonViewController(title: "Coming soon..")
-                comingSoonController.view.backgroundColor = DesignSystem.shared.colors.surfacePrimary.default
-                comingSoonController.tabBarItem.image = UIImage(systemName: "pencil.and.scribble")
+                let moreNavigationController = UINavigationController()
+                let moreViewController = More.build(navigationController: moreNavigationController)
+                moreNavigationController.viewControllers = [moreViewController]
```

Comment:
`moreNavigationController` is not needed since we already have a `navigationController`